### PR TITLE
Fixes github URLs and removes comment signatures

### PR DESCRIPTION
**Pull request by abea**

### CODE REVIEW

File: docs/core-concepts/editable-content-on-pages/custom-widgets.md [hunk 1]
```diff
@@ -106,7 +106,7 @@ mkdir -p lib/modules/page-link-widgets/views
 ```
 
 ::: tip
-While it's good to get some experience making all the folders and files yourself, the `apostrophe-cli` pacakage that you installed earlier can easily create basic module structures for you from templates. You can learn more in the [`apostrophe-cli` README](https://github.com/apostrophecms/apostrophe-cli/blob/master/README.md).
+While it's good to get some experience making all the folders and files yourself, the `apostrophe-cli` package that you installed earlier can easily create basic module structures for you from templates. You can learn more in the [`apostrophe-cli` README](https://github.com/apostrophecms/apostrophe-cli/blob/master/README.md).
```
Comment (localghost443) [inline docs/core-concepts/editable-content-on-pages/custom-widgets.md:109]: Again I question this. I would almost reverse this logic.

Comment (localghost443) [reply docs/core-concepts/editable-content-on-pages/custom-widgets.md:109]: monkey see monkey do

File: docs/core-concepts/editable-content-on-pages/layout-widgets.md [hunk 2]
```diff
@@ -8,7 +8,7 @@ You want a widget that creates a layout of inner widgets.
 
 Generally, you need a dedicated widget to create layout. The Apostrophe team sometimes refers to these as ‘layout widgets’ that give form and structure to ‘content widgets’ \(widgets whose sole job is to display content\). **In this example we'll make a simple two column layout.**
```
Comment (localghost443) [inline docs/core-concepts/editable-content-on-pages/layout-widgets.md:9]: "You can use widgets to create layouts -- we like to call them..."

File: docs/devops/multicore.md [hunk 3]
```diff
@@ -173,7 +173,7 @@ modules: {
 
 By default Apostrophe writes uploaded files to the server's local hard drive. This is no good if you are load balancing across multiple servers.
 
-Fortunately Apostrophe uses [uploadfs](https://github.com/punkave/uploadfs), which also supports using Amazon S3 as a back end for file storage.
+Fortunately Apostrophe uses [uploadfs](https://github.com/apostrophecms/uploadfs), which also supports using Amazon S3 as a back end for file storage.
```
Comment (localghost443) [inline docs/devops/multicore.md:176]: 'uploadfs is well supported by.....'

File: docs/reference/modules/apostrophe-areas/browser-apostrophe-areas-editor.md [hunk 4]
```diff
@@ -153,7 +153,7 @@ Override default apostrophe-context functionality because we
 need to use $.onSafe, for the sake of nested areas.
 ### on(*eventType*, *selector*, *fn*)
 This is a wrapper for $.onSafe that avoids events that are actually
-happening in nested areas. -Tom and Sam
```
Comment (localghost443) [inline docs/reference/modules/apostrophe-areas/browser-apostrophe-areas-editor.md:156]: LOL

File: docs/core-concepts/editable-content-on-pages/custom-widgets.md (hunk 1 above)
Comment (boutell) [reply docs/core-concepts/editable-content-on-pages/custom-widgets.md:109]: Not everything can be done with the CLI. Some understanding of the structure is both good and necessary.

Comment (abea) [reply docs/core-concepts/editable-content-on-pages/custom-widgets.md:109]: @localghost443 Could you flag this for discussion in the CLI branch? And I'm not 100% clear on what you're suggesting.

**Pull request merged**